changelog.d/gh-1058.fixed

```diff
@@ -0,0 +1,35 @@
+Taint analysis: track `ptr->field` l-values in C++
+
+In C++, we now track tainted field access via pointer dereference. For
+instance, consider the following code snippet:
+```
+void test_intra_001() {
+  TestObject *obj = new TestObject();
+
+  obj->a = taint_source();
+  obj->b = SAFE_STR;
+
+  // ok: cpp-tainted-field-ptr
+  sink(obj->b, __LINE__);
+  // ruleid: cpp-tainted-field-ptr
+  sink(obj->a, __LINE__);
+}
+```
+
+This can be matched by the rule:
```

Comment:
In these toy examples for illustration purposes I think it's fine to skip the rule, it doesn't add much value but makes the changelog a lot longer. If we want to provide something for the user to reproduce, I think it's better a Playground permalink.